web/html/src/manager/virtualization/guests/list/guests-list.js

```diff
@@ -0,0 +1,401 @@
+const React = require("react");
+const PropTypes = React.PropTypes;
+const MessageContainer = require("../../../../components/messages").Messages;
+const MessagesUtils = require("../../../../components/messages").Utils;
+const {Table, Column, SearchField, Highlight} = require("../../../../components/table");
+const Functions = require("../../../../utils/functions");
+const Utils = Functions.Utils;
+const {LinkButton, AsyncButton} = require("../../../../components/buttons");
```

Comment:
After https://github.com/uyuni-project/uyuni/pull/143 you can refactor all these `../../../../components/buttons` to just `components/buttons`